ci/test_python_dask.sh

```diff
@@ -5,11 +5,12 @@
 source "$(dirname "$0")/test_python_common.sh"
 
 rapids-logger "pytest cuml-dask"
+pushd python/cuml/tests/dask
```

Comment:
Does this need a corresponding `popd`? Or are you trying to make the last command `pytest` so its exit code is returned? If so, just use `cd`. Same in the other file.

Reply:
I'll add a commit to switch to `cd`. I think it's just remnants of when these two test files were consolidated.

Reply:
done in f355f1c